### PR TITLE
fix(docs): remove optional mark from kafka's computed fields

### PR DESCRIPTION
**Pull request by byashimov**
## About this change—what it does

Fixes documentation for kafka.

### CODE REVIEW

File: docs/resources/kafka.md [hunk 1]
```diff
@@ -75,6 +74,7 @@ resource "aiven_kafka" "kafka1" {
 - `disk_space_step` (String) The default disk space step of the service, possible values depend on the service type, the cloud provider and the project. `disk_space` needs to increment from `disk_space_default` by increments of this size.
 - `disk_space_used` (String) Disk space that service is currently using
 - `id` (String) The ID of this resource.
+- `kafka` (List of Object) Kafka server provided values (see [below for nested schema](#nestedatt--kafka))
```
Comment (byashimov) [inline docs/resources/kafka.md:77]: I couldn't fix this. It just ignores maxItems.

File: docs/resources/kafka.md [hunk 2]
```diff
@@ -313,6 +301,18 @@ Read-Only:
 - `ssl` (Boolean)
 - `usage` (String)
 
+
+<a id="nestedatt--kafka"></a>
+### Nested Schema for `kafka`
+
+Read-Only:
+
+- `access_cert` (String)
```
Comment (Serpentiel) [inline docs/resources/kafka.md:310]: it no longer says `Sensitive`

Comment (byashimov) [reply docs/resources/kafka.md:310]: But I can't make it use that word. Or can I?

Comment (Serpentiel) [reply docs/resources/kafka.md:310]: why doesn't it works anymore 🤔

**Comment by Serpentiel** [comment]
also looks like unit tests are [failing](https://github.com/aiven/terraform-provider-aiven/actions/runs/4895987847/jobs/8742232105?pr=1152#step:4:98)

**Pull request merged**